app/src/main/java/io/github/droidkaigi/confsched2018/presentation/NavigationController.kt

```diff
@@ -107,4 +110,21 @@ class NavigationController @Inject constructor(private val activity: AppCompatAc
     fun navigateToTopicDetailActivity(topicId: Int) {
         TopicDetailActivity.start(activity, topicId)
     }
+
+    fun navigateToExternalBrowser(url: String) {
+        val intent = CustomTabsIntent.Builder()
+                .setShowTitle(true)
+                .setToolbarColor(ContextCompat.getColor(activity, R.color.primary))
+                .setExitAnimations(
+                        activity,
+                        android.R.anim.slide_in_left,
+                        android.R.anim.slide_out_right
+                )
```

Comment:
Just suggestion.
How about adding the referrer for the analytics? Like this.

```kotlin
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
            customTabsIntent.intent.putExtra(Intent.EXTRA_REFERRER, Uri.parse("android-app://" + activity.packageName))
        }
```

Reply:
Sounds good :)
Will address that asap!

Reply:
oops, sorry. I forgot to address this. Let me fix quickly~

Reply:
Fixed https://github.com/DroidKaigi/conference-app-2018/pull/176/commits/ca47cc5b6cbb76ba86da8849c08768bd34c4b6f2.
JFYI, minSdk is greater than jelly bean 💯